src/UglyToad.PdfPig/Content/Letter.cs

```diff
@@ -1,171 +1,176 @@
-﻿namespace UglyToad.PdfPig.Content
-{
-    using Core;
-    using Graphics.Colors;
+﻿namespace UglyToad.PdfPig.Content

```

Comment:
Not sure what happened here (Formating tabs/whitespace?). Only added interface and

```
        /// <summary>
        /// Gets the Bounding Box: The rectangle completely containing this object
        /// </summary>
        public PdfRectangle BoundingBox => GlyphRectangle;
```